refinery/tool_manager/models.py

```diff
@@ -932,6 +994,18 @@ def _get_workflow_step(self, galaxy_dataset_dict):
         )
         return workflow_steps[0]
 
+    def _has_dataset_collection_input(self):
+        """
+        Determine whether our Workflow's input step has a Galaxy Dataset
+        Collection as input or not.
+        :returns: Boolean
+        """
+        workflow_input_type = self._get_workflow_dict()["steps"]["0"]["type"]
+        if workflow_input_type == self.DATA_COLLECTION_INPUT:
+            return True
+        else:
+            return False
```

Comment:
It could just be 
```
return workflow_input_type == self.DATA_COLLECTION_INPUT
```
but going as far as
```
return self._get_workflow_dict()["steps"]["0"]["type"] == self.DATA_COLLECTION_INPUT
```
is probably less clear?